server/rooms.js

```diff
@@ -1323,8 +1325,9 @@ class BasicChatRoom extends BasicRoom {
 	onUpdateIdentity(user) {
 		if (user && user.connected) {
 			if (!this.users[user.userid]) return false;
+			const statusText = user.getStatus() ? `@${user.getStatus()}` : '';
```

Comment:
Should `getStatus` just always include the `@`?

Reply:
I didn't mostly because the `@` isn't actually part of the status message, it's just part of the protocol.

Reply:
`getStatus` has been removed.